controllers/suite_test.go

```diff
@@ -52,11 +52,12 @@ import (
 // http://onsi.github.io/ginkgo/ to learn more about Ginkgo.
 
 var (
-	k8sClient client.Client
-	testEnv   *envtest.Environment
-	ctx       context.Context
-	cancel    context.CancelFunc
-	log       logr.Logger
+	k8sClient           client.Client
+	testEnv             *envtest.Environment
+	ctx                 context.Context
+	cancel              context.CancelFunc
+	log                 logr.Logger
+	dependenciesUpdater *TestComponentDependenciesUpdater
```

Comment:
Values here are global for all tests. please move controller specific stuff to the controller itself.